css/styles.css

```diff
@@ -18,11 +18,20 @@ body {
   border-radius: 50%;
   max-width: 50%;
   height: auto;
+  align-self: center;
+}
+
+.profile {
+  display: flex;
+  flex-direction: column;
 }
 
 .tech-stack__list {
   list-style: none;
   padding-left: 0;
+  display: flex;
+  gap: 10px;
+  flex-wrap: wrap;
```

Comment:
You can add `justify-content: space-between;` to place the `<li>` elements.

Reply:
> You can add `justify-content: space-between;` to place the `<li>` elements.

As an alternative to "gap"? Yes, that`s possible, bust as the idea is to create gaps of at least 10px, "gap" works fine as well, right?

Reply:
True👌